website/docs/examples/file_reader_can/README.md

```diff
@@ -5,8 +5,18 @@ title: File Reader CAN
 
 # File Reader CAN Example
 
-This [**file_reader_can example**](https://github.com/farm-ng/farm-ng-amiga/tree/main-v2/py/examples/file_reader_can/main.py)
-parses the CAN data from a recorded log and prints the `AmigaTpdo1` parsed
+:::info Basic Knowledge Requirements
+
+Before diving into this example, here's a quick overview of what you'll need to be familiar with:
+
+1. **Python Programming**: This code is written in Python.
+Basic constructs such as functions, conditional statements, loops, and more are utilized.
+
+2. **Canbus Concepts**: We also recommend you check our [Canbus Service webpage](../../concepts/canbus_service)
```

Comment:
Same comment about being consistent between all examples as for GPS. Also note the way we do relative links (no `../` as that is more error prone)

```suggestion
2. **[farm-ng Canbus Service Overview](/docs/concepts/canbus_service/)**:
This overview provides a base understanding of the gRPC service the client you create will connect to.
```